packit_service/service/api/usage.py

```diff
@@ -0,0 +1,543 @@
+              https://github.com/packit/ogr: 34
+              https://github.com/packit/packit: 54
+        top_projects_by_job_runs:
+          https://github.com/cockpit-project/cockpit: 6937
+          https://github.com/packit/hello-world: 10409
+          https://github.com/systemd/systemd: 14489
+      sync_release_runs:
+        job_runs: 419
+        per_event:
+          branch_push:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+          issue:
+            job_runs: 22
+            top_projects_by_job_runs:
+              https://github.com/martinpitt/python-dbusmock: 3
+              https://github.com/packit/packit: 3
+              https://github.com/packit/specfile: 6
+          pull_request:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+          release:
+            job_runs: 397
+            top_projects_by_job_runs:
+              https://github.com/martinpitt/python-dbusmock: 35
+              https://github.com/packit/packit: 35
+              https://github.com/rhinstaller/anaconda: 34
+        top_projects_by_job_runs:
+          https://github.com/martinpitt/python-dbusmock: 38
+          https://github.com/packit/packit: 38
+          https://github.com/rhinstaller/anaconda: 34
+      tft_test_run_targets:
+        job_runs: 150525
+        per_event:
+          branch_push:
+            job_runs: 441
+            top_projects_by_job_runs:
+              https://github.com/oamg/convert2rhel: 209
+              https://github.com/packit-service/packit: 50
+              https://github.com/python-bugzilla/python-bugzilla: 88
+          issue:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+          pull_request:
+            job_runs: 150026
+            top_projects_by_job_runs:
+              https://github.com/cockpit-project/cockpit: 21157
+              https://github.com/oamg/convert2rhel: 15297
+              https://github.com/teemtee/tmt: 22136
+          release:
+            job_runs: 58
+            top_projects_by_job_runs:
+              https://github.com/fedora-infra/fedora-messaging: 8
+              https://github.com/fedora-iot/zezere: 8
+              https://github.com/psss/tmt: 21
+        top_projects_by_job_runs:
+          https://github.com/cockpit-project/cockpit: 21157
+          https://github.com/oamg/convert2rhel: 15506
+          https://github.com/teemtee/tmt: 22136
+      vm_image_build_targets:
+        job_runs: 2
+        per_event:
+          branch_push:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+          issue:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+          pull_request:
+            job_runs: 2
+            top_projects_by_job_runs:
+              https://github.com/packit/ogr: 2
+          release:
+            job_runs: 0
+            top_projects_by_job_runs: {}
+        top_projects_by_job_runs:
+          https://github.com/packit/ogr: 2
```

Comment:
docstring length: 🥇

:D but it's great that one can exactly see the format and sample data